Successfully updated:
Created Frame node DbzuqOuiK
Created Frame node ncNR_YkJq
Created Text node WPEe2ILif
Created Text node yYK86UExb
Created Frame node kEtdV23gu
Updated attributes for node DbzuqOuiK
Updated attributes for node ncNR_YkJq
Updated text for node WPEe2ILif
Updated attributes for node WPEe2ILif
Updated text for node yYK86UExb
Updated attributes for node yYK86UExb
Updated attributes for node kEtdV23gu
Reordered node ncNR_YkJq within parent DbzuqOuiK to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3398,20 +3398,73 @@
       Header Text
     </HeaderText>
   </Frame>
   <BodyContentGoesHere
       nodeId="fXcbul4zf"
       width="fit-content"
       height="fit-content"
       font="Inter"
   >
     Body content goes here
   </BodyContentGoesHere>
   <Frame
       nodeId="URe1o6R8_"
       width="100%"
       height="1fr"
       <!-- background color string or project color style path (if starts with /) -->
       backgroundColor="rgb(255, 255, 255)"
       borderRadius="4px"
    />
 </Stack>
+<Stack
+    nodeId="DbzuqOuiK"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="ncNR_YkJq"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="WPEe2ILif"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="yYK86UExb"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="kEtdV23gu"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.